requirements.txt

```diff
@@ -1,4 +1,4 @@
-ovh==0.5.0
+ovh==1.0.0
 djehouty==0.1.5
 schedule==1.0
-requests==2.25.1
```

Comment:
The `ovh` package [already brings `requests`](https://github.com/ovh/python-ovh/blob/master/setup.cfg#L32-L33), so there's no need to re-specify it with another version.